ui/components/multichain/multi-srp/import-srp/index.ts

```diff
@@ -0,0 +1 @@
+export { ImportSrp as ImportSRP } from './import-srp';
```

Comment:
Are we using `ImportSRP` still elsewhere? We should not need this alias IMO.

```suggestion
export { ImportSrp as ImportSRP } from './import-srp';
```